contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
```

Comment:
There's a lot of inconsistencies in the terminating a line with a semi-colon. Would be good for every line to use one, or not use one.

Might be good to add linting, but I guess another PR?